dali/operators/decoder/nvjpeg/decoupled_api/nvjpeg_decoder_decoupled_api.h

```diff
@@ -23,10 +23,13 @@
 #include <memory>
 #include <numeric>
 #include <atomic>
-
+#include <mutex>
```

Comment:
Maybe you can include in cc file only?

Reply:
removed, it was not needed here any more